packages/esm-appointments-app/src/form/appointments-form.component.tsx

```diff
@@ -350,6 +352,13 @@ const AppointmentsForm: React.FC<AppointmentsFormProps> = ({
       <InlineLoading className={styles.loader} description={`${t('loading', 'Loading')} ...`} role="progressbar" />
     );
 
+  const updateLocations = uniqBy(
```

Comment:
Where is this used?

Reply:
The `updateLocations` function is defined in the AppointmentsForm component but is not directly used within the component itself. We can use it to optimize performance by fetching and updating the list only when necessary, rather than fetching it every time the component renders.( Its for the future purposes if its relative static)